pages/workflow.md

```diff
@@ -1,54 +1,107 @@
+{% include nav.html %} {% include css.html %}
+
 # Workflow
+	
+The information used for the visualization was gathered by using 
```

Comment:
```suggestion
The information used for the visualization was gathered by using the data provided by [**www.slavevoyages.org**](https://slavevoyages.org) and after a bit of clean up that allowed the generated 
data to work with other programs the workflow became more manageable.
```

Reply:
Watch out, you had what you meant to be one paragraph split between multiple lines. This can cause some funky stuff to happen. I have fixed it here, and below.

Also, link when you put any URL or online or digital project in the text. Link www.slavevoyages.org. Even if you say the Slave Voyages project, you should still link that. Link Palladio as well.